R/add_transition_risk_category_at_company_level.R

```diff
@@ -0,0 +1,14 @@
+add_transition_risk_category_at_company_level <- function(data) {
+  data |>
+    rename(grouped_by = "benchmark_tr_score", risk_category = "transition_risk_category") |>
+    epa_at_company_level() |>
+    insert_row_with_na_in_risk_category() |>
+    rename(benchmark_tr_score_avg = "grouped_by", transition_risk_category = "risk_category", transition_risk_category_share = "value")
```

Comment:
Maybe the calls to `rename()` should move to an `adapt_*()` of some kind. The task of calculating risk categories is higher-level and different to the lower-level task of adapting the data from one format to the other (i.e. renaming column names in this case).

Reply:
Thanks! refactored :)